cypress/integration/chi-vue/search-input.spec.js

```diff
@@ -0,0 +1,207 @@
+const SEARCH_INPUTS_DATA_CY = {
+  BASE: '[data-cy="base-search-input-test"]',
+  DISABLED: '[data-cy="disabled-search-input-test"]',
+  ATTRS: '[data-cy="attrs-search-input-test"]',
+  DELETE: '[data-cy="delete-value-search-input-test"]',
+  EVENTS: '[data-cy="chi-events-trigger-search-input-test"]'
+};
+const ELEMENTS = {
+  BUTTON: 'button',
+  DIV: 'div',
+  ICON: 'i',
+  INPUT: 'input'
+};
+const COMPONENT_CLASSES = {
```

Comment:
Are we checking `chi-form__item`, `chi-search__input`classes, `-icon--right` classes?

Reply:
Now yes, solved.